tuf-spec.md

```diff
@@ -173,7 +173,9 @@ repo](https://github.com/theupdateframework/specification/issues).
       of the problem.
 
       + **Malicious mirrors preventing updates.**  A repository mirror cannot
-      prevent updates from good mirrors.
+      prevent updates from good mirrors. This includes mirrors that have been
```

Comment:
Could you elaborate on how bad mirrors could prevents updates from good mirrors (i.e., what are the plausible mechanisms)?

Also, do you mean good mirrors by those "that have been taken down due to" etc?